content/en/docs/architecture/step-registry.md

```diff
@@ -645,6 +645,15 @@ declared in its `env` section, but _will not_ be set in
 added to that step as well. Note that this case evades the unused parameter
 validation since at least one step declares that it uses the relevant parameter.
 
+### Additional Fields
+
+There are few extra fields that can be configured in order to control if and when the test must be executed.
+-`run_if_changed` Set a regex to make the job trigger only when a pull request changes a certain path in the repository (see the [upstream doc](https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#triggering-jobs-based-on-changes)).
+-`skip_if_only_changed` Set a regex to skip triggering the job when all the changes in the pull request match (see the [upstream doc](https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#triggering-jobs-based-on-changes)).
+-`optional` Set to `true` to `make` the job not block merges.
+
+**Note:** `run_if_changed` and `skip_if_only_changed` are mutually exclusive.
+
```

Comment:
These fields have nothing to do with registry, there is no reason why this should be in the registry doc.

IMO this content needs to go under items in https://docs.ci.openshift.org/docs/architecture/ci-operator/#types-of-tests . I'd probably add the descriptions under presubmits, and just added a small note to postsubmits that they can set run_if/skip_if but not optional.